WTWSMS/common/religions/00_religions.txt

```diff
@@ -1527,6 +1540,7 @@ muslim = {
 	}
 	druze = {
 		graphical_culture = muslimgfx
+		has_coa_on_barony_only = yes
```

Comment:
What do you mean? I hope that you are aware that the wording "kebab" to refer to Turks is considered as racial slur and not allowed on the Pi forums?